clang/lib/DPCT/ASTTraversal.cpp

```diff
@@ -2555,6 +2556,40 @@ void TypeInDeclRule::runRule(const MatchFinder::MatchResult &Result) {
       }
     }
 
+    if (CanonicalTypeStr == "cooperative_groups::__v1::thread_group") {
+     // Skip migrate the type in function body.
```

Comment:
Can we merge the code with the cooperative_groups::__v1::thread_block_tile< and cooperative_groups::__v1::thread_block